packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -3875,6 +3875,8 @@ void main() {
     ));
 
     assert(focusNode.hasFocus);
+    // Autofocus has an one frame delay.
+    await tester.pump();
```

Comment:
Before the change the underline was shown even when the text field is not focused. That's why this test was passing without the additional `pump`.